websites/C/CHub/presence.ts

```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({
+		clientId: "1000715027994640404",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const { host } = document.location,
+		title = document.title.split("-");
+	let { title: dhname } = document,
+		{ pathname: path } = document.location;
+	path = path.replaceAll("/nighty", "");
+	if (title.length === 2) dhname = title[1].replace(/^[.\s]+|[.\s]+$/g, "");
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/Ut1y4n3.png",
+		smallImageText: "chub.page",
+		details: dhname,
+		buttons: [{ label: "View Drivers Hub", url: document.URL }],
```

Comment:
Static buttons are not allowed

Reply:
It's not a static button?

Reply:
> @EncryptedDev It's not a static button?

It is, almost all buttons are static except 2 (delivery log and member).

I admit that this presence doesn't meet your guidelines, and the team see no need of creating a presence on PreMiD, that's why I closed this PR.